playbooks/install_stack.yaml

```diff
@@ -227,6 +283,8 @@
       tripleo_deploy_output_dir: "{{ ansible_env.HOME }}"
       tripleo_deploy_local_ip: "{{ public_api }}"
       tripleo_deploy_control_virtual_ip: "{{ control_plane_ip }}"
+      #TODO(emilien) figure out if we need this:
```

Comment:
So, is it needed or not?

Reply:
so it doesn't do do anything, the public endpoint still is on the control plane IP... I'll investigate later